emsdk.py

```diff
@@ -2775,7 +2770,14 @@ def extract_bool_arg(name):
 
   arg_old = extract_bool_arg('--old')
   arg_uses = extract_bool_arg('--uses')
+  arg_permanent = extract_bool_arg('--permanent')
   arg_global = extract_bool_arg('--global')
+  if arg_global:
+    print('--global is deprecated. Use `--system` to set the environment variables for all users')
+  arg_system = extract_bool_arg('--system')
+  if arg_system:
+    arg_permanent = True
+    arg_global = True
```

Comment:
Did you mean to set arg_global here?  I would imagine `arg_global` should no longer be used anywhere after this change?

Reply:
Fixed in https://github.com/emscripten-core/emsdk/pull/621/commits/bd028b9b6278e6e7b58b972c11de62ce3b0e3748